operator/v1/types_ingress.go

```diff
@@ -332,6 +340,10 @@ type IngressControllerStatus struct {
 	//     * DNS records have been successfully created.
 	//   - False if any of those conditions are unsatisfied.
 	Conditions []OperatorCondition `json:"conditions,omitempty"`
+
+	// tlsProfile is the TLS connection configuration that is in effect.
+	// +optional
+	TLSProfile *configv1.TLSProfileSpec `json:"tlsProfile,omitempty"`
```

Comment:
What about reflecting the TLS profile details in status? Something like:
```
TLSProfile *configv1.TLSProfileSpec `json:"tlsProfileSpec,omitempty"`
```

Reply:
Done